src/main/java/com/google/sps/servlets/ElectionServlet.java

```diff
@@ -0,0 +1,100 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.UnsupportedEncodingException;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLEncoder;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonSyntaxException;
+
+@WebServlet("/election")
+public class ElectionServlet extends HttpServlet {
+
+  private static final String baseURL = "https://www.googleapis.com/civicinfo/v2/elections?key=";
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        StringBuilder strBuf = new StringBuilder();  
+        
+        HttpURLConnection conn = null;
+        BufferedReader reader = null;
+
+        try {  
+            URL url = new URL(baseURL + "AIzaSyBlNN_uUrbqY2uRaBpKTD-5Jbw6nk8n0_k");
```

Comment:
I think Julian asked but I missed the response -- is this an API key? If so, I think it's not supposed to be on our Github repo

Reply:
As far as I understand, this key is unique to our Cloud Shell project; that being said I saw a way to “hide” the API in a different file that doesn’t go onto github and will try that.

Reply:
I restricted the API key to the Google Civic Information API and this URL specifically:
![image](https://user-images.githubusercontent.com/38237443/88257906-b351a880-cc73-11ea-8eac-0f1be46de8fd.png)

Reply:
I looked into this, and I was able to limit our API key for use with the Civic Information API only, but can only limit to use with our website once our website is launched (otherwise, we would need to reset an env variable every time this program ran). For the sake of hiding the API key regardless of use limitations, I made it an environment variable that both you and @gianelgado12 will have to store in your .bashrc files under the name GOOGLE_API_KEY. If we decide to limit the API key to just our website, we won't need to change any code, and the API key is now hidden just in case.